### PR TITLE
Moved the metadata into setup.cfg

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

**Comment by shapiromatron** [comment]
@KOLANICH if you can get the tests to pass in py36 we can accept this PR - are there updates that can be made that would fix this?

**Comment by KOLANICH** [comment]
Done.

**Comment by shapiromatron** [comment]
Sorry, will be merging things or closing them this week. I dont understand how to version this library anymore with this method - is there a command I have to type?

**Comment by KOLANICH** [comment]
>I dont understand how to version this library anymore with this method - is there a command I have to type?

Just create a git tag first folloving the format `v<version number>` (i.e. `v0.6.1`). It is possible to do it via a CLI, but I usually use GUI frontends for git - TortoiseGit on Windows, gitk on Linux, you also may want to look at gitahead. Then build a wheel using `python3 -m build -nwx .` (`python3 setup.py bdist_wheel` should also work, but `build` is now the recommended way) and upload it to pypi using `twine`. If version is still not set correctly, the first thing you should try is to upgrade `setuptools` and optionally all the other tools (modern enough versions of setuptools should upgrade them themselves, but I recommend to just install the latest versions of the tools, this is the first thing I do when I install python on a machine).

**Comment by shapiromatron** [comment]
Ok, I'm going to try this out and may add some docs to describe. Seems like a good approach if I can figure it out... thanks!

File: rispy/__init__.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 """Read and write RIS and WOK files"""
 
-__version__ = "0.6.0"
-
+from .version import __version__  # noqa
```
Comment (shapiromatron) [inline rispy/__init__.py:3]: getting a merge conflict; to resolve I made these changes: https://github.com/MrTango/rispy/blob/pr/34/rispy/__init__.py
Screen Shot 2021-05-24 at 3 59 03 PM

**Review by shapiromatron — CHANGES_REQUESTED** [review]
Getting a few merge conflicts after the latest merges; I dont think I can push to your branch - would you please fix?

Hoping to mint a new version in the next few days if you have the time...  I'll merge it this time I promise :)

**Comment by shapiromatron** [comment]
@KOLANICH think you might have a chance today to fix? I'm going to release tomorrow I hope?

File: rispy/__init__.py (hunk 1 above)
Comment (KOLANICH) [reply rispy/__init__.py]: Done.

File: Makefile [hunk 2]
```diff
@@ -43,7 +43,7 @@ clean:  ## remove all build, test and Python artifacts
 	find . -name '__pycache__' -exec rm -fr {} +
 
 lint:  ## Check for python formatting issues via black & flake8
-	@black . --check && flake8 .
+	@black . --check && flake8 ./rispy && flake8 ./tests
```
[the remark this answers is not in the document]
Comment (KOLANICH) [reply Makefile]: Done.

**Review by shapiromatron — APPROVED** [review]

**Comment by shapiromatron** [comment]
ci failure; add .eggs and build to https://github.com/MrTango/rispy/blob/master/pyproject.toml#L5

**Pull request merged**